apps/src/fish/locale-do-not-import.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * DO NOT IMPORT THIS DIRECTLY. Instead do:
+ *   ```
+ *   import msg from '@cdo/fish/locale'.
+ *   ```
+ * This allows the webpack config to determine how locales should be loaded,
+ * which is important for making locale setup work seamlessly in tests.
+ */
+// locale for fish
```

Comment:
I guess a similar comment exists in other similar files, but I don't quite know what it means...